components/Common/Breadcrumbs/index.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import { type LinkProps } from 'next/link';
+import { useMemo, type FC } from 'react';
+
+import LocalizedLink from '@/components/LocalizedLink';
+
+import BreadcrumbItem from './BreadcrumbItem';
+import BreadcrumbRoot from './BreadcrumbRoot';
+import styles from './index.module.css';
+
+type BreadcrumbLink = {
+  label: string;
+  href: LinkProps['href'];
+};
+
+type BreadcrumbsProps = {
+  links: BreadcrumbLink[];
+  maxLength?: number;
+  hideHome?: boolean;
+};
+
+const Breadcrumbs: FC<BreadcrumbsProps> = ({
+  links,
+  maxLength = 3,
+  hideHome = false,
+}) => {
+  const items = useMemo(
+    () =>
+      (links.length > maxLength
```

Comment:
For the sake of keeping the code clean, it does make sense to assign this to a constant,

also, you could simply do this:

```ts

const itemsToRender = links.slice(links.length > maxLength && maxLength || 0);
```

For all situations, this logic should work. If maxLength